docs/fuse-box/partners/grove.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: How to use Grove with Fuse in 3 easy steps.
+sidebar_position: 3
+---
+
+[Grove](https://www.grove.city) streamlines connecting to the decentralized web, offering developers seamless blockchain interaction right from the start. This quickstart guide is your roadmap to acquiring an endpoint, enabling your projects to communicate with Fuse Mainnet networks effortlessly. Let’s dive into the steps:
+
+
+## Step 1: Sign Up
+[Sign up](https://portal.grove.city) for a Grove account.
+
+
+## Step 2: Create Your Application
+Creating an application is your first step towards active interaction with the Fuse Blockchain.
+
+<img
+  src="/img/grove/grove-dashboard-one.png"
+  width="100%"
+  height="50%"
+/>
+
+On the Grove Dashboard, you will find an already created **Hello** Project in the left navigation bar. Right under it, you will see the “New Application” Button.
```

Comment:
Hey, i followed your guide and when i first create account i didnt had any created projects, please fix this.

![Screenshot 2024-04-30 at 14 58 35](https://github.com/fuseio/fuse-docs/assets/68244817/21413a46-53cf-401c-a3af-8c298fac4f80)

Reply:
Fixed.